### ISSUE TITLE
Provide access to the pull request's message

### ISSUE THREAD

**Issue by nex3** [travis-ci/travis-ci#9302]
It's useful to be able to send messages to build infrastructure via the pull request message—things like `[ci skip]`, but with semantics specific to the project at hand. It would be really useful if, for pull request builds, Travis provided access to the pull request message as an environment variable, file on disk, or something similar.

It's possible to get this via the GitHub API, and in practice this is what we do in the [sass-spec repo](https://github.com/sass/sass-spec/blob/master/tools/skipped-for-impl.sh#L17). However, this is very flaky; we regularly run into GitHub rate limits which prevent the logic from working correctly. Since Travis already has access to the information about the pull request, it would be great if it could provide that information to the build.

**Comment by BanzaiMan** [comment]
Hello. Am I correct to assume that you mean "title" of the PR, rather than "message", that you want access to? (https://developer.github.com/v3/pulls/#get-a-single-pull-request)

**Comment by BanzaiMan** [comment]
If so, this is a duplicate of #9288.

**Comment by nex3** [comment]
No, we need access to what the GitHub API refers to as the "body". For example, in https://github.com/sass/sass-spec/pull/1229, we want to be able to look for `[skip dart-sass]` and skip tests accordingly.

**Comment by BanzaiMan** [comment]
Thanks for the clarification.

**Comment by nex3** [comment]
Not to be a bother, but we're still seeing a *lot* of flakes that this would let us avoid.

**Comment by nevercast** [comment]
Is there any intent to visit this in the near future? I tool would like to add CI instructions in the body of the PR request to not dirty up the PR title.

**Comment by nex3** [comment]
This is still causing substantial trouble for the Sass project.

**Comment by fjahr** [comment]
This would be very helpful for me as well, much more than the title.
